visualization/app/codeCharta/ui/nodeContextMenu/flattenButtons/flattenButtons.component.html

```diff
@@ -0,0 +1,13 @@
+<button
+	mat-button
+	*ngIf="!codeMapNode?.isFlattened"
+	(click)="flattenNode()"
+	title="Flatten node and children-nodes, but keep an empty space"
+>
+	<img alt="flattenIcon" class="png-icon" src="../../../../assets/flatten.png" style="width: 17px; height: 17px" />
+	FLATTEN
```

Comment:
```suggestion
	Flatten
```

Reply:
Are you sure we want to change it? Previously the AngularJS md-button made all text upper case by default. This would be the only lower case entry now
![image](https://user-images.githubusercontent.com/3596742/151539507-630b1bb3-424a-4eb9-9b19-7a681346fd00.png)